x-pack/legacy/plugins/siem/server/lib/detection_engine/rules/read_rules.test.ts

```diff
@@ -47,6 +106,20 @@ describe('read_rules', () => {
       expect(rule).toEqual(getResult());
     });
 
+    test('should return null if the output from alertsClient with ruleId set is empty', async () => {
+      const alertsClient = alertsClientMock.create();
+      alertsClient.get.mockResolvedValue(getResult());
+      // @ts-ignore
+      alertsClient.find.mockResolvedValue({ data: [] });
```

Comment:
I really would avoid `ts-ignore` at all if possible. It is a really terrible thing to use as only last resort.

In this unit test you can either do the `unknown` cast trick or just flush it out normally:

```ts
alertsClient.find.mockResolvedValue({ data: [], page: 0, perPage: 1, total: 0 });
```

I would flush it out normally as a regular test case.

Also you can just use the typical find result mock which already returns an empty value:

```ts
alertsClient.find.mockResolvedValue(getFindResult());
```

It's just poorly named as `getFindResult()` when really it returns an empty find.